static/app/components/smartSearchBar/searchDropdown.tsx

```diff
@@ -70,22 +72,31 @@ function SearchDropdown({
               <Fragment key={item.title}>
                 {item.type === 'header' && <HeaderItem group={item} />}
                 {item.children &&
-                  item.children.map(child => (
-                    <DropdownItem
-                      key={getDropdownItemKey(child)}
-                      item={child}
-                      searchSubstring={searchSubstring}
-                      onClick={onClick}
-                      onIconClick={onIconClick}
-                      additionalSearchConfig={{
-                        ...getSearchConfigFromCustomPerformanceMetrics(
-                          customPerformanceMetrics
-                        ),
-                        supportedTags,
-                      }}
-                      customInvalidTagMessage={customInvalidTagMessage}
-                    />
-                  ))}
+                  item.children.map(child => {
+                    if (
+                      mergeSearchGroupWith &&
```

Comment:
wonder if we want to include `title` in the prop name... `mergeWithTitle`? idk, not sure how to name this.

Reply:
i suppose it depends on the context. internally the SmartSearchBar calls them `searchGroups` by the time it propagates to `SearchDropdown` its items so technically props should be `<SmartSearchBar mergeSearchGroupWith={{}} />` -> `<SearchDropdown mergeItemsWith={{}} />`

🤷🏼